pkg/cmd/pr/view/view.go

```diff
@@ -260,19 +261,19 @@ func formattedReviewerState(cs *iostreams.ColorScheme, reviewer *reviewerState)
 		state = commentedReviewState
 	}
 
-	var colorFunc func(string) string
+	var displayState string
 	switch state {
 	case requestedReviewState:
-		colorFunc = cs.Yellow
+		displayState = cs.Yellow("Requested")
 	case approvedReviewState:
-		colorFunc = cs.Green
+		displayState = cs.Green("Approved")
 	case changesRequestedReviewState:
-		colorFunc = cs.Red
+		displayState = cs.Red("Changes requested")
 	default:
-		colorFunc = func(str string) string { return str } // Do nothing
+		displayState = text.Title(state)
```

Comment:
Does it make sense to titleize an already uppercase string? We could just leave the `default` clause return the machine-readable identifier for unknown states in the future so that we know we should maybe handle them in the UI.

Reply:
I think if we want to change this behavior we should do it in a PR explicitly for changing it rather than in this one which is meant to maintain current behavior.